### ISSUE TITLE
ElementAccessExpressions should be vertically indented

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1543]
Similar to MiKo_6040, element access expressions such as following should be reported:

```c#
var descendant = child.Children[1]
    .Children[2]
    .Children[3]
    .Children[4];
```

They should be fixed into:

```c#
var descendant = child.Children[1]
                      .Children[2]
                      .Children[3]
                      .Children[4];
```
